vars/withZalenium.groovy

```diff
@@ -119,7 +119,7 @@ void waitForSeleniumToGetReady(String host) {
 boolean isSeleniumReady(String host) {
     sh(returnStdout: true,
             script: "curl -sSL http://${host}:4444/wd/hub/status || true") // Don't fail
-            .contains('status\": 0')
+            .contains('ready\": true')
```

Comment:
Nice catch!

Did you check the output? Ready == true vs the hub is capable of starting new workers?

Reply:
Ready is only true when there are enough idle worker nodes connected to the grid hub.